Makefile

```diff
@@ -127,6 +127,18 @@ test-go: ## Run tests for backend.
 	@echo "test backend"
 	$(GO) test -v ./pkg/...
 
+test-go-unit: ## Run unit tests for backend.
+	@echo "test backend unit tests"
+	$(GO) test -short -covermode=atomic -timeout=30m ./pkg/...
+
+test-go-integration: ## Run integration tests for backend.
+	@echo "test backend integration tests"
+	$(GO) test -run Integration -covermode=atomic -timeout=30m ./pkg/...
+
+test-go-pro: ## Run enterprise2 pro tests for backend.
+	@echo "test backend enterprise2 pro tests"
+	$(GO) test -tags=pro -covermode=atomic -timeout=30m ./pkg/...
```

Comment:
Pro shouldn't be here, it won't work if you don't have enterprise linked in the repo.

Reply:
we did use to have a test enterprise, where did that disappear?